src/main/webapp/WEB-INF/jsp/preview.jsp

```diff
@@ -66,7 +66,7 @@
             <div class="col-sm-12">
                 <c:if test="${showConfigLink}">
                     <portlet:renderURL var="configUrl" portletMode="CONFIG"/>
-                    <a class="btn btn-primary pull-right" role="button" href="${ configUrl }"><i class="fa fa-gear" aria-hidden="true"></i> <spring:message code="preview.configure"/></a>
+                    <a class="btn btn-primary pull-right" role="link" href="${ configUrl }"><i class="fa fa-gear" aria-hidden="true"></i> <spring:message code="preview.configure"/></a>
```

Comment:
I believe that the `<a>` tag has `role="link"` by default.

Reference: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_link_role